boolebots/src/components/BotPanel/botpanel.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../styles/global.scss";
+@import "../../styles/variables.scss";
```

Comment:
You could have imported only the `global.scss` because this already imports the  `variable.scss`.